.gitignore

```diff
@@ -20,6 +20,11 @@
 # Ignore config
 /config/database.yml
 
+# Ignore SETUP_USE_CUSTOM files
+/config/database.mysql.yml.custom
+/.env.bootstrap.custom
+/db/seeds.custom.yml
```

Comment:
not a fan of custom ignores ... how about putting it into `tmp/` or something like `custom/` and then ignore that ?

Reply:
Yes, either one will work fine. tmp/ or custom/.

Reply:
I removed the custom gitignore entries.